File: system-test/java/src/test/java/com/optivem/shop/systemtest/legacy/mod06/smoke/external/TaxSmokeTest.java
package com.optivem.shop.systemtest.legacy.mod06.smoke.external;

import com.optivem.shop.systemtest.legacy.mod06.base.BaseChannelDriverTest;
import org.junit.jupiter.api.Test;

import static com.optivem.shop.dsl.common.ResultAssert.assertThatResult;

class TaxSmokeTest extends BaseChannelDriverTest {
    @Test
    void shouldBeAbleToGoToTax() {
        var result = taxDriver.goToTax();
        assertThatResult(result).isSuccess();
    }
}

File: system-test/dotnet/SystemTests/Legacy/Mod06/SmokeTests/External/TaxSmokeTest.cs
using Common;
using SystemTests.Legacy.Mod06.Base;
using Shouldly;
using Xunit;

namespace SystemTests.Legacy.Mod06.SmokeTests.External;

public class TaxSmokeTest : BaseChannelDriverTest
{
    [Fact]
    public async Task ShouldBeAbleToGoToTax()
    {
        var result = await _taxDriver!.GoToTaxAsync();
        result.ShouldBeSuccess();
    }
}
